pages/services/edge-lb/1.0.0/installing/index.md

```diff
@@ -85,13 +85,35 @@ Use the following CLI commands to provision the Edge-LB service account with the
 
 All CLI commands can also be executed via the [IAM API](/latest/security/ent/iam-api/).
 
-1.  Grant the permissions and the allowed actions to the service account using the following commands. The commands below allow your Edge-LB service to manage DC/OS packages, Marathon tasks, Edge-LB pools and tasks.
+1. For easier administration, add `edge-lb-principal` to the `superusers` group
```

Comment:
Prior to the list, I would provide the two options as a list for example: To securely administer Edge-LB one of two methods can be used: 
1. Create a superuser account
2. Create a edge-lb administrator account that is limited to performing edge-lb related tasks.

The above wording isn't perfect, but I'm hoping that I get my thoughts across.